src/core.c

```diff
@@ -126,7 +126,31 @@ SEXP ToggleFullscreen_R(void)
 
 
 
+// Drawing-related functions
 
+SEXP BeginDrawing_R(void)
```

Comment:
Functions need to be declared in a header file (core.h). It works now because no other file (translation unit) uses these functions. But these need to be added to `init.c` Call method list so R will make these functions available to `.Call()`